src/Aspire.Hosting.Analyzers/Aspire.Hosting.Analyzers.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
I think this line can be removed since this csproj doesn't support packing - IsPackable=false.

Reply:
Copied from dotnet/aspnetcore https://github.com/dotnet/aspnetcore/blob/843f7b011dae5687c77f3570542325b01301f8db/src/Framework/AspNetCoreAnalyzers/src/Analyzers/Microsoft.AspNetCore.App.Analyzers.csproj#L4-L7

Reply:
This is different. You copied some properties, and then set other properties that nullify these copied properties.

Reply:
Ah OK, yes `<IsPackable>false</IsPackable>` came from the Roslyn analyzer template IIRC. We're structuring this a bit different to how the template does so very likely we can simplify this a bit.